app/src/test/java/jp/developer/retia/frozenword/TestExt.kt

```diff
@@ -0,0 +1,41 @@
+package jp.developer.retia.frozenword
+
+import kotlinx.coroutines.*
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.test.TestCoroutineDispatcher
+import kotlinx.coroutines.test.resetMain
+import kotlinx.coroutines.test.setMain
+import org.spekframework.spek2.dsl.LifecycleAware
+
+@ExperimentalCoroutinesApi
+fun LifecycleAware.setMainDispatcher(dispatcher: TestCoroutineDispatcher = TestCoroutineDispatcher()) {
```

Comment:
```suggestion
fun LifecycleAware.setMainDispatcher(
    dispatcher: TestCoroutineDispatcher = TestCoroutineDispatcher()
) {
```



<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>